frontend/packages/enduser-frontend/src/config.ts

```diff
@@ -179,7 +179,7 @@ configs._default = {
     daycareApplication: true,
     preschoolApplication: true,
     selectApplicationType: true,
-    attachments: true
+    attachments: false
```

Comment:
@hruoho this wasn't necessary

Reply:
i'm lost

Reply:
😂

Reply:
You changed it so that attachments are hidden _by default_ instead of just in _prod_.

Reply:
I see :facepalm: